tests/lua/spectre_common_test.lua

```diff
@@ -646,18 +650,19 @@ describe("spectre_common", function()
                 {['X-Smartstack-Destination'] = 'srv.main'} -- request_headers
             )
 
-            assert.are.equal(500, resp.status)
-            assert.are.equal('Error requesting /bar: there was an error. Whoops.', resp.body)
+            assert.are.equal(200, resp.status)
+            assert.are.equal(false, resp.no_response)
+            assert.are.equal('RESULT', resp.body)
         end)
 
-        it("Forwards body when HTTP request succeeds", function()
+        it("Errors out when the downstream HTTP request returns an error", function()
```

Comment:
Do you think its worth covering the -1 case as well here?

Reply:
So the following test covers the "no response" scenario, which triggers the -1 log / header:
https://github.com/Yelp/casper/blob/98ad7e145831382120bb6f3f7c1342ec6fa493c9/tests/lua/spectre_common_test.lua#L681

Reply:
👍